README.rst

```diff
@@ -89,16 +89,43 @@ Releasing a new version
 Here are the steps you need to follow in order to push a new release:
 
 * Make sure that ``HISTORY.rst`` is up-to-date focusing mostly on backwards incompatible changes.
+
+  Security vulnerabilities should be clearly marked in a "Security issues" section along with
+  a level indicator of:
+
+  * High: vulnerability facilitates data loss, data access, impersonation of admin, or allows access
+    to other sites or components
+
+    Users should upgrade immediately.
+
+  * Medium: vulnerability endangers users by sending them to malicious sites or stealing browser
+    data.
+
+    Users should upgrade immediately.
+
+  * Low: vulnerability is a nuissance to site staff and/or users
+
+    Users should upgrade.
```

Comment:
I kind of made this up based on looking at CVE severity calculator (https://www.first.org/cvss/calculator/3.0) and some other things.

Mostly, I think we minimally need a "LOW" which indicates security issues that are more of a nuissance and "HIGH" which indicates security issues that require users to upgrade asap.

I don't want to make this a huge complex thing, but I figured it was helpful to provide guidance to release folks.

What do you think? Are these levels and descriptions helpful? Do they line up with other projects/systems?